src/api/ledger/pathfind.js

```diff
@@ -46,8 +46,11 @@ function requestPathFind(remote, pathfind: PathFind, callback) {
       _.omit(utils.common.toRippledAmount(amount), 'value'));
   }
 
-  remote.requestRipplePathFind(params,
-    composeAsync(_.partial(addParams, params), callback));
+  remote.createPathFind(params, data => {
+    if (data.full_reply) {
+      callback(null, addParams(params, data));
```

Comment:
This `callback ` might be called more than once? cause the stream hasn't been told to be closed ?